packages/core/src/engine/Autodiff.ts

```diff
@@ -939,6 +939,9 @@ const emitGraph = (
   return vals;
 };
 
+/** TODO: make this not a global */
+const memory = new WebAssembly.Memory({ initial: 0 });
```

Comment:
My understanding is you want to pass this around as long as you are on the same "thread," like it wouldn't make sense for each diagram to have their own memory instance? 

If that's the case, it seems a little tricky to expose this all the way at the `core` API level. Maybe it's really okay to leave it as a global?

Reply:
Yeah pretty much. So if you want it to stay a global, can you just go ahead and take the justification you wrote here and turn it into a code comment (replacing the "TODO")? Then I'm cool to merge this.